src/libs/EmojiUtils.js

```diff
@@ -110,6 +110,23 @@ function trimEmojiUnicode(emojiCode) {
     return emojiCode.replace(/(fe0f|1f3fb|1f3fc|1f3fd|1f3fe|1f3ff)$/, '').trim();
 }
 
+/**
+ * Validates first character is emoji in text string
+ *
+ * @param {String} message
+ * @returns {Boolean}
+ */
+function firstLetterIsEmoji(message) {
```

Comment:
I would prefer this as `isFirstLetterEmoji` or `isFirstCharacterEmoji`

Reply:
done, please check.